src/current/cockroachcloud/billing-management.md

```diff
@@ -51,6 +62,30 @@ If you used a CockroachDB {{ site.data.products.dedicated }} trial code while [c
 Your credit card will be charged after the trial ends.
 {{site.data.alerts.end}}
 
+</section>
+
+<section class="filter-content" markdown="1" data-scope="marketplace">
+
+## Subscribe through AWS Marketplace
+
+To subscribe to CockroachDB {{ site.data.products.cloud }} through the AWS Marketplace:
+
+1. Navigate to the [CockroachDB product page](https://aws.amazon.com/marketplace/pp/prodview-3zbkzekdohwly) on the AWS Marketplace and click **View purchase options**.
+1. On the **Subscribe to CockroachDB Cloud** page, click **Subscribe** and then click **Set up your account**. You will be directed to the CockroachDB {{ site.data.products.cloud }} console.
+1. [Register]({% link cockroachcloud/create-an-account.md %}#register-a-new-account) a new CockroachDB {{ site.data.products.cloud }} account or sign in to your existing account.
+1. Select the organization CockroachDB {{ site.data.products.cloud }} you want to subscribe through the AWS Marketplace and click **Subscribe to AWS Marketplace**.
```

Comment:
Maybe consider adding a disclaimer here because not all your orgs will be eligible for connecting to the AWS Marketplace. 

Orgs already linked to a marketplace subscription and invoice orgs are ineligible. You must also have the Billing Coordinator role in the org to link it.

Reply:
It might be good to highlight eligibility requirements, as this is one area a user may get confused. We have a error message in the UI that says you can read more in the docs about eligibility